eva_submission/evapro/populate_evapro.py

```diff
@@ -0,0 +1,454 @@
+            eva_species_name = eva_species_name or common_name or scientific_name
+            taxonomy_obj = Taxonomy(taxonomy_id=taxonomy_id, common_name=common_name, scientific_name=scientific_name,
+                                    taxonomy_code=taxonomy_code, eva_name=eva_species_name)
+            self.eva_session.add(taxonomy_obj)
+            self.info(f'Add Taxonomy {taxonomy_id} to EVAPRO')
+        return taxonomy_obj
+
+
+    def insert_linked_projects(self, project_obj, linked_project_accession, project_relation='PARENT'):
+        query = select(LinkedProject).where(LinkedProject.project_accession == project_obj.project_accession,
+                                            LinkedProject.linked_project_accession == linked_project_accession,
+                                            LinkedProject.linked_project_relation == project_relation)
+        result = self.eva_session.execute(query).fetchone()
+        if result:
+            linked_project_obj = result.LinkedProject
+        else:
+            linked_project_obj = LinkedProject(project_accession=project_obj.project_accession,
+                                               linked_project_accession=linked_project_accession,
+                                               linked_project_relation=project_relation)
+            self.eva_session.add(linked_project_obj)
+        return linked_project_obj
+
+    def insert_ena_submission(self, ena_submission_accession, action, submission_alias, submission_date, brokered=True, submission_type='PROJECT'):
+        query = select(Submission).where(Submission.submission_accession == ena_submission_accession)
+        result = self.eva_session.execute(query).fetchone()
+        if result:
+            submission_obj = result.Submission
+        else:
+            submission_obj = Submission(
+                submission_accession=ena_submission_accession, action=action, title=submission_alias, date=submission_date, brokered=brokered, type=submission_type
+            )
+            self.eva_session.add(submission_obj)
+            self.info(f'Add Submission {ena_submission_accession} {action} to EVAPRO')
+        return submission_obj
+
+    def insert_project_ena_submission(self, project_obj, submission_obj, eload):
+        '''
+        $sth_eva = $dbh_eva->prepare("insert into PROJECT_ENA_SUBMISSION (PROJECT_ACCESSION,SUBMISSION_ID) values (?,?)");
+        foreach my $sub_id(@{$accession_to_id->{'SUBMISSION_ID'}}){
+            $sth_eva->execute($project_accession,$sub_id);
+        }
+        $sth_eva = $dbh_eva->prepare("insert into EVA_SUBMISSION (EVA_SUBMISSION_ID,EVA_SUBMISSION_STATUS_ID) values (?,?)");
+        $sth_eva->execute($eload,6);
+        $sth_eva = $dbh_eva->prepare("insert into PROJECT_EVA_SUBMISSION (PROJECT_ACCESSION,old_ticket_id,ELOAD_ID) values (?,?,?)");
+        $sth_eva->execute($project_accession,$eload,$eload);
+        '''
```

Comment:
I'd remove this and any other perl code, unless there are any bits you haven't implemented yet and need to keep track of